### PR TITLE
axes fix for loading JPG with no exif_tags

### PR DESCRIPTION
**Pull request by ThorstenBASF**
### Description of the change
A few sentences and/or a bulleted list to describe and motivate the change:
- JPG files with no exif_tags had no axes or dimensions when loaded.
- instead of returning empty axes, axes are returned with an scale of 1 pixel/pixel.
- with fix this images can be handled with hs/axes commands as usual. (s.calibrate e.g.)

### Progress of the PR
- [ ] Change implemented (can be split into several points),
- [ ] update docstring (if appropriate),
- [ ] update user guide (if appropriate),
- [x] add a changelog entry in the `upcoming_changes` folder (see [`upcoming_changes/README.rst`](https://github.com/hyperspy/rosettasciio/blob/main/upcoming_changes/README.rst)),
- [ ] Check formatting of the changelog entry (and eventual user guide changes) in the `docs/readthedocs.org:rosettasciio` build of this PR (link in github checks)
- [ ] add tests,
- [x] ready for review.
---
- [x] add testfile

### Minimal example of the bug fix or the new feature
```python
import hyperspy.api as hs
tfile = "./rsciio/tests/data/image/jpg_no_exif_tags.jpg"
s = hs.load(tfile)
s.calibrate(x0=0, y0=0, x1=0, y1=1, new_length=0.2, units="nm", interactive=False)
```

### CODE REVIEW

File: rsciio/utils/image.py [hunk 1]
```diff
@@ -41,13 +41,18 @@
...
     if exif_tags is None:
-        return []
-    offsets = exif_tags.get("FocalPlaneXYOrigins", [0, 0])
-    # jpg files made with Renishaw have this tag
-    scales = exif_tags.get("FieldOfViewXY", [1, 1])
+        # if no exif_tags exist, axes are set to scale of 1 pixel/pixel
+        # return of axes must not be empty, or dimensions are lost
+        offsets = [0,0]
+        scales = [sizes[1],sizes[0]]
+        unit = "pixel"
```
Comment (ericpre) [inline rsciio/utils/image.py:48]: In this case, the standard pattern is to set to None and in hyperspy will be parsed a `traits.api.undefined` value.
```suggestion
        unit = None
```

File: rsciio/utils/image.py [hunk 2]
```diff
@@ -41,13 +41,18 @@
...
 def _parse_axes_from_metadata(exif_tags, sizes):
     if exif_tags is None:
-        return []
-    offsets = exif_tags.get("FocalPlaneXYOrigins", [0, 0])
-    # jpg files made with Renishaw have this tag
-    scales = exif_tags.get("FieldOfViewXY", [1, 1])
+        # if no exif_tags exist, axes are set to scale of 1 pixel/pixel
+        # return of axes must not be empty, or dimensions are lost
+        offsets = [0,0]
+        scales = [sizes[1],sizes[0]]
```
Comment (ericpre) [inline rsciio/utils/image.py:47]: While you are at it and in the interest of readability, can you please rename here and in the two other places below to 
```suggestion
        fiels_of_views = [sizes[1],sizes[0]]
```

**Review by ericpre — COMMENTED** [review]
Thanks, this looks good, I left two comments.

File: rsciio/utils/image.py (hunk 2 above)
Comment (ThorstenBASF) [reply rsciio/utils/image.py]: Thanks, did the changes, also set unit to "None".

**Comment by ericpre** [comment]
pre-commit.ci autofix

**Review by ericpre — APPROVED** [review]

**Pull request merged**